### ISSUE TITLE
Terminal bell and garbled interface every few track changes on Kitty and rxvt-unicode.

### ISSUE THREAD

**Issue by featherfurl** [clangen/musikcube#400]
Every few times musikcube loads a new track either automatically through a playlist or through manual selection it will ring the terminal bell & print the file path of the new track + some garbled characters. On kitty this often completely garbles the interface and stops it from refreshing when interacted with, and on rxvt-unicode it prints a message in a warning overlay that has to be manually clicked before it disappears, but does not interrupt the interface.

It seems like some wrong characters are occasionally being inserted into the command to play the next track and this is causing issues. I really like musikcube and would like to continue using it, but this issue is getting a bit painful.

I'm running arch linux + i3wm and can provide screenshots if necessary.

**Comment by clangen** [comment]
Yeah, a couple other folks have reported similar issues. It's very frustrating for me because I'm having a really hard time reproducing the problem.

Can you try the following and see if it helps your particular issue?
1. Upgrade to 0.96.3 (required)
2. Open the musikcube `settings` screen
3. Navigate over to `advanced settings`
4. Set `DisableWindowTitleUpdates` to `true`

**Comment by featherfurl** [comment]
So I just did this and clicking through a heap of manual track changes doesn't seem to trigger the issue. I thought I'd tried toggling every setting in the advanced settings menu before posting so I might run it a big longer and see if anything happens on a longer session. It's entirely possible I just thought I'd toggled the window title option and run it for a while when I had not actually done so. Either way it's worth having information on this problem recorded somewhere for others because I couldn't find anything about it when I first looked online.
